### PR TITLE
travis build button

### PR DESCRIPTION
**Pull request by smwatts**

### CODE REVIEW

**Review by apchiodo — APPROVED** [review]

File: vignettes/vignette.Rmd [hunk 1]
```diff
@@ -33,6 +33,7 @@ Now that the package is loaded, you will have access to three functions.
 Let's start with the stdizer function!
 
 ```{r}
+library(gapminder)
 # Load gapminder data set
 df <- gapminder::gapminder
```
Comment (apchiodo) [inline vignettes/vignette.Rmd:38]: `library(gapminder)` shouldn't be necessary if we have `gapminder::gapminder` call.

Comment (jchen9314) [reply vignettes/vignette.Rmd:38]: There is an error in Travis: 

Error: processing vignette 'vignette.Rmd' failed with diagnostics:
there is no package called 'gapminder'

**Comment by apchiodo** [comment]
image

**Comment by apchiodo** [comment]
Things we need to fix before travis works.  Probably. ^^^

**Comment by smwatts** [comment]
Fixing it now

**Comment by jchen9314** [comment]
In .travis.yml
We might need to add the following:

```
r_packages:
    - gapminder
```

**Comment by apchiodo** [comment]
See #31 .

**Pull request closed without merging**